content/en/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -254,6 +254,9 @@ indicates one of the following reasons for the Pod termination:
 `DeletionByPodGC`
 : Pod, that is bound to a no longer existing Node, is due to be deleted by [Pod garbage collection](/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection).
 
+`TerminationByKubelet`
+: Pod is being terminated by Kubelet (either due to {{<glossary_tooltip term_id="node-pressure-eviction" text="Node Pressure Eviction">}} or [Graceful Node Shutdown](/docs/concepts/architecture/nodes/#graceful-node-shutdown)) and is due to be deleted by the controller owning the Pod.
```

Comment:
```suggestion
: Kubelet terminates the Pod, because of the {{<glossary_tooltip term_id="node-pressure-eviction" text="Node Pressure Eviction">}} or the [Graceful Node Shutdown](/docs/concepts/architecture/nodes/#graceful-node-shutdown). Pod is due to be deleted by the controller owning the Pod.
```